config/environments/production.rb

```diff
@@ -84,8 +84,14 @@
   # Do not dump schema after migrations.
   config.active_record.dump_schema_after_migration = false
 
-  config.abr_host = 'https://alwaysberunning.net'
-  config.nrdb_api_host = 'https://api.netrunnerdb.com'
+  config.abr_host = ENV['ABR_HOST']
```

Comment:
It looks like this environment variable didn't exist before. Should we default the ABR host to what it was before?